projects/fsastorefrontstyles/scss/components/checkout/_quote-review.scss

```diff
@@ -0,0 +1,6 @@
+%cx-fs-quote-review {
+  .message-position {
```

Comment:
Since we already have:
```
  .top {
    top: 0;
  }
  .bottom {
    bottom: 0;
  }
  .right {
    right: 0;
  }
```
defined in `%cx-fs-enriched-responsive-banner`, maybe we can move it to `building-blocks.scss` and have it as global classes.

Reply:
Yes, it would be nice.